packages/data/tests/unit/helpers/metric-format-test.js

```diff
@@ -17,6 +17,9 @@ test('It formats metrics appropriately', function(assert) {
   assert.equal(metricFormat({metric: {name: 'foo', longName: 'Foo'}, parameters: {p: 1}}, 'Bar'),
     'Bar (1)',
     'Formats metric with parameters');
+  assert.equal(metricFormat({ metric: { name: 'foo', longName: 'Foo' }, parameters: { p: { defaultValue:1 } } }, 'Bar'),
+    'Bar (1)',
+    'Formats metric with parameters');
```

Comment:
@Xuefeng-Zhu Can we update this msg to be more specific than the msg above?

Reply:
updated